src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -233,7 +233,7 @@ private async Task StartChainAsync()
             ChainedHeader chainTip = await this.chainRepository.LoadAsync(this.chain.Genesis).ConfigureAwait(false);
             this.chain.SetTip(chainTip);
 
-            this.logger.LogInformation("Chain loaded at height " + this.chain.Height);
+            this.logger.LogInformation("Chain loaded at height {0}", this.chain.Height);
```

Comment:
.

Reply:
Well spotted! .. the {0} that was missing ;)